core/src/datayoga_core/blocks/map/tests/test_map.py

```diff
@@ -0,0 +1,123 @@
+import pytest
+from datayoga_core import result
+from datayoga_core.blocks.map.block import Block
+
+
+@pytest.mark.asyncio
+async def test_map_expression_jmspath():
```

Comment:
let's rename to:
`test_map_expression_jmespath`

Reply:
done